docs/getting-started.md

```diff
@@ -203,7 +203,7 @@ sudo curl -X PUT --unix-socket "${API_SOCKET}" \
     }" \
     "http://localhost/logger"
 
-KERNEL="./vmlinux-5.10.210"
+KERNEL="./$(ls vmlinux*)"
```

Comment:
what if we have multiple files matching `vmlinux*`? can we use the last one (a newer version prioritized).

```suggestion
KERNEL="./$(ls vmlinux*  | tail -1)"
```

Reply:
Makes sense, changed it in 575dc907f29d9f2859875773c3f2a43ba69d6b92

Reply:
Could you squash them into one commit and add commit message to make gitlint rule pass?
https://buildkite.com/firecracker/firecracker-pr/builds/10551#0191235b-06c3-4d68-a65d-cd4f969d4256/62-182

Just FYI: you can run style checks by running `tools/devtool checkstyle` before submission (https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L428-L429)

Reply:
@markusz thank you for your quick fix! I kicked a new CI run :) Let's wait for it to end.